JWT auth client configuration example in README is inaccurate

#### Environment details

  - OS: MacOS Mojave 10.14.6 (18G2022)
  - Node.js version: lts/erbium
  - npm version: 6.13.4
  - `google-auth-library` version: 6.0.0

#### Steps to reproduce

  1. Install Google Auth Library
  2. Attempt to authorize service client via JWT example in [readme](https://github.com/googleapis/google-auth-library-nodejs#json-web-tokens).
  3. Receive type error: `src/__tests__/google.ts:44:7 - error TS2345: Argument of type 'null' is not assignable to parameter of type 'string | undefined'.`

The example should probably be using the `JWTOptions` constructor rather than the multi-parameter one.

Making sure to follow these steps will guarantee the quickest resolution possible.

Thanks!

--- a/README.md
+++ b/README.md
@@ -250,12 +250,11 @@ const {JWT} = require('google-auth-library');
 const keys = require('./jwt.keys.json');
 
 async function main() {
-  const client = new JWT(
-    keys.client_email,
-    null,
-    keys.private_key,
-    ['https://www.googleapis.com/auth/cloud-platform'],
-  );
+  const client = new JWT({
+    email: keys.client_email,
+    key: keys.private_key,
+    scopes: ['https://www.googleapis.com/auth/cloud-platform'],
+  });
   const url = `https://dns.googleapis.com/dns/v1/projects/${keys.project_id}`;
   const res = await client.request({url});
   console.log(res.data);

